test/snippet/nanocourse/bwt_occurrence_table2.cpp

```diff
@@ -45,30 +45,30 @@ std::string bwt_construction(std::string const & text)
 struct occurrence_table
 {
     // The list of bitvectors:
-    std::vector<Bitvector> bitvector_data;
+    std::vector<Bitvector> data;
 
     // We want that 5 bitvectors are filled depending on the bwt,
     // so let's customise the constructor of occurrence_table:
     occurrence_table(std::string const & bwt)
     {
         // resize the 5 bitvectors, blocks and superblocks:
-        bitvector_data = std::vector<Bitvector>(5, Bitvector((bwt.size() + 63)/ 64));
+        data = std::vector<Bitvector>(5, Bitvector((bwt.size() + 63)/ 64));
 
         // fill the bitvectors
         for (size_t i = 0; i < bwt.size(); ++i)
         {
             switch (bwt[i])
             {
-                case '$': bitvector_data[0].write(i, 1); break;
-                case 'i': bitvector_data[1].write(i, 1); break;
-                case 'm': bitvector_data[2].write(i, 1); break;
-                case 'p': bitvector_data[3].write(i, 1); break;
-                case 's': bitvector_data[4].write(i, 1); break;
+                case '$': data[0].write(i, 1); break;
+                case 'i': data[1].write(i, 1); break;
+                case 'm': data[2].write(i, 1); break;
+                case 'p': data[3].write(i, 1); break;
+                case 's': data[4].write(i, 1); break;
                 default: break;
             }
         }
 
-        for (Bitvector & bitv : bitvector_data)
+        for (Bitvector & bitv : data)
             bitv.construct(3, 12);
```

Comment:
The sizes are still hard coded. Fine for now?

Reply:
Yes. Good catch though, I added instructions for the sizes in the tutorial.

Reply:
Yes. Good catch though, I added instructions for the sizes in the tutorial.

Reply:
Should now be 3 and 6 though?